forgerock-openbanking-uk-aspsp-common/src/test/java/com/forgerock/openbanking/common/services/frequency/FrequencyServiceTest.java

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright 2019 ForgeRock AS.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.forgerock.openbanking.common.services.frequency;
+
+import com.forgerock.openbanking.common.model.openbanking.frequency.FrequencyType;
+import com.forgerock.openbanking.common.services.openbanking.frequency.FrequencyService;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Unit test for {@link com.forgerock.openbanking.common.services.openbanking.frequency.FrequencyService}.
```

Comment:
I like the fact you added the class under test (partly because it makes it quick and easy to click through).

Minor - it doesn't need to be fully qualified.

Reply:
done!